### ISSUE TITLE
Doesn't render in Chrome behind SSL

### ISSUE THREAD

**Issue by nadeemja** [apendua/meteor-mathjax#14]
Hello,

I've found that Meteor-mathjax fails to render (goes blank) when using SSL/HTTPS with Chrome. 

Chrome without SSL works well. 

Other browser work with SSL too.

Please help. Really urgent.

**Comment by apendua** [comment]
@nadeemja Can you provide some error logs?

**Comment by apendua** [comment]
@nadeemja Which version of Chrome exactly are you using?

**Comment by nadeemja** [comment]
Dear @apendua,

It turned out to be the SSL configuration of CloudFlare that was problematic.
Switching from "Full (strict)" to "Flexible" solved the issue. 

The only error was something about "A Parser-blocking, cross-origin script," in the console.
A yellow warning. It was related too a cloudflare.min.js file. I'm using the latest version of Chrome for mac. 53.0.2785.116 (64-bit)

I'm most grateful that you replied so soon. Thank you very much. :)

**Comment by apendua** [comment]
@nadeemja Do you this issue is now solved?

**Comment by nadeemja** [comment]
Dear @apendua, 

Unfortunately, I was wrong. The SSL is not itself the problem. Rather it's some optimisation that CloudFlare does. SSL in Chrome is working now, however, I've had to turn off CloudFlare for now.

Have you experimented with CloudFlare?

**Comment by apendua** [comment]
@nadeemja Yes, I was using CloudFlare in at least two project and I've never had major problems with their services. One important thing when you're using CloudFlare with a Meteor project is to make sure that the DDP traffic from your application bypasses CloudFlare entirely. Here's a good blog-post explaining the problem and how it can be solved:

https://meteorhacks.com/cloudflare-meets-meteor/

**Comment by nadeemja** [comment]
Thank you for the suggestion, @apendua.
Recently, CloudFlare added support for DDP and it works fine.

I'll alert the CloudFlare of this as well. Perhaps they have some idea.

When was the last time you used CloudFlare with meteor-mathjax?
